src/main/java/dev/openfeature/sdk/Metadata.java

```diff
@@ -5,4 +5,8 @@
  */
 public interface Metadata {
     String getName();
```

Comment:
I think we should deprecate this now (@toddbaert  appreciate your opinion)

```suggestion
     @Deprecated
    default String getName(){
        return "";
    }
```

Reply:
Agreed